bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormComponent.java

```diff
@@ -76,6 +76,15 @@ default Map<String, Object> getDorProperties() {
         return Collections.emptyMap();
     }
 
+    /**
+     * Template Policy key-value Custom Properties
+     */
+    @JsonView(Views.Author.class)
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    default Map<String, String> getHeadlessCustomProperties() {
```

Comment:
headless sounds a bit restrictive to me. headful should also be able to leverage these custom props. the CSR may also be owned by adobe in some cases.

Reply:
`getCustomProperties` should be good IMHO. It would cover any property which is not in OOTB list.

Reply:
`getCustomProperties` is already an existing function, you should only use it